stdlib/public/Platform/tgmath.swift.gyb

```diff
@@ -82,6 +93,25 @@ public func frexp<T: BinaryFloatingPoint>(_ x: T) -> (T, Int) {
   return (x.significand / 2, Int(x.exponent + 1))
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+@usableFromInline
+func _vjpSqrt<T: FloatingPoint & Differentiable> (
+  _ x: T
+) -> (T, (T) -> T) where T == T.TangentVector {
+  let value = x.squareRoot()
+  return (value, {v in (1 / 2) * ( 1 / value) * v})
+}
+
+@usableFromInline
+func _vjpFma<T: FloatingPoint & Differentiable> (
+  _ x: T,
+  _ y: T,
+  _ z: T
+) -> (T, (T) -> (T, T, T)) where T == T.TangentVector {
+  return (fma(x, y, z),
+    {v in return (v * y, v * x, v)})
```

Comment:
* Move this to the end of the previous line.
* Add a space after each `{` before each `}`. Same for all other occurrences.
* Omit `return` when a closure has a single expression.

Reply:
done in 6b1ebf264b6801e5becf1b9f5d6206ce627af1f6